assets/prefabs/Cranberry/Cranberry.prefab

```diff
@@ -5,9 +5,16 @@
   },
   "Item": {
     "icon": "AdditionalFruits:Cranberry",
-    "stackId": "cranberry"
+    "stackId": "cranberry",
```

Comment:
While you are editing these files - could you please align all these `stackId`s to be aligned with `"AdditionalFruits:BlueberryBush"`, please?

Reply:
All of the stackIDs have been adjusted to the same form 👍